src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $day-block-size * 7 + $gap-size * 6 + $side-padding * 2;
```

Comment:
What are 7, 6, 2? Try to avoid magic numbers. Also, would be great to create a separate variable for calendar width and move all calculations there